### PR TITLE
couple funcs

### PR DESCRIPTION
**Pull request by FazanaJ**
just getting it off local

### CODE REVIEW

File: src/object_functions.c [hunk 1]
```diff
@@ -3950,6 +3950,11 @@ void obj_init_weapon(Object *obj, UNUSED LevelObjectEntry_Weapon *entry) {
     obj->unk7C.word = 0;
 }
 
+/**
+ * Racer weapon loop behaviour.
+ * Rockets will move forwards, or towards their victim.
+ * Statiuonary traps will drop to the floor and stay there for a while before self-detonating.
```
Comment (Ryan-Myers) [inline src/object_functions.c:3956]:
```suggestion
 * Stationary traps will drop to the floor and stay there for a while before self-detonating.
```

File: src/object_functions.c [hunk 2]
```diff
@@ -4115,9 +4120,121 @@ void handle_rocket_projectile(Object *obj, s32 updateRate) {
     }
 }
 
+/**
+ * If a collision target is found for a level 1 or 3 rocket, reverse the velocity and set the next position target
+ * to be right on top of them. This ensures a collision is garunteed this frame.
```
Comment (Ryan-Myers) [inline src/object_functions.c:4125]:
```suggestion
 * to be right on top of them. This ensures a collision is guaranteed this frame.
```

File: src/object_functions.c [hunk 3]
```diff
@@ -4115,9 +4120,121 @@ void handle_rocket_projectile(Object *obj, s32 updateRate) {
...
+            obj->segment.trans.x_rotation = arctan2_f(diffY, dist);
+            obj->segment.trans.y_rotation = angle;
+        }
+    }
+    play_rocket_trailing_sound(obj, rocket, SOUND_INCOMING_ROCKET);
+}
+
+/**
+ * If a collision target is found for a level 2 rocket, reverse the velocity and set the next position target
+ * to be right on top of them. This ensures a collision is garunteed this frame.
```
Comment (Ryan-Myers) [inline src/object_functions.c:4172]:
```suggestion
 * to be right on top of them. This ensures a collision is guaranteed this frame.
```

File: src/objects.c [hunk 4]
```diff
@@ -92,12 +92,21 @@ s16 D_800DC7B8[52] = {
...
+    {VEHICLE_CAR,           VEHICLE_TRICKY}, // Tricky 2
+    {VEHICLE_HOVERCRAFT,    VEHICLE_BLUEY}, // Bluey 2
+    {VEHICLE_HOVERCRAFT,    VEHICLE_BUBBLER}, // Bubbler 1
+    {VEHICLE_HOVERCRAFT,    VEHICLE_BUBBLER}, // Bubbler 2
+    {VEHICLE_PLANE,         VEHICLE_SMOKEY}, // Smokey 2
+    {VEHICLE_CAR,           VEHICLE_WIZPIG}, // Wizpig 1
+    {VEHICLE_PLANE,         VEHICLE_ROCKET}, // Wizpig 2
 };
+
+s8 D_800DC820[10] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
```
Comment (Ryan-Myers) [inline src/objects.c:108]: Unused?

**Review by Ryan-Myers — APPROVED** [review]
LGTM

File: src/objects.c (hunk 4 above)
Comment (FazanaJ) [reply src/objects.c:108]: it's a split, so purpose currently unknown, but definitely used.

**Pull request merged**